docs/tutorials/03_quantum_kernel.ipynb

```diff
@@ -354,11 +353,7 @@
    "source": [
     "adhoc_feature_map = ZZFeatureMap(feature_dimension=adhoc_dimension, reps=2, entanglement=\"linear\")\n",
     "\n",
-    "adhoc_backend = QuantumInstance(\n",
-    "    BasicAer.get_backend(\"qasm_simulator\"), shots=1024, seed_simulator=seed, seed_transpiler=seed\n",
-    ")\n",
-    "\n",
-    "adhoc_kernel = QuantumKernel(feature_map=adhoc_feature_map, quantum_instance=adhoc_backend)"
+    "adhoc_kernel = FidelityQuantumKernel(feature_map=adhoc_feature_map)"
```

Comment:
The para. text a little below still refers to `QuantumKernel` in the second sentence

>The scikit-learn spectral clustering algorithm allows us to define a [custom kernel] in two ways: by providing the kernel as a callable function or by precomputing the kernel matrix. Using the QuantumKernel class in qiskit, we can only use the latter.

Reply:
Replaced with "to define a custom kernel in two ways: by providing the kernel as a callable function or by precomputing the kernel matrix. Using the `FidelityQuantumKernel` class in Qiskit Machine Learning,"